core/src/program/blocks/span_block.rs

```diff
@@ -38,21 +39,26 @@ const MAX_OPS_PER_BATCH: usize = GROUP_SIZE * BATCH_SIZE;
 pub struct Span {
     op_batches: Vec<OpBatch>,
     hash: Digest,
+    decorator_map: DecoratorMap,
```

Comment:
I would probably just call this `decorators`

Reply:
Done